py/h2o_wave/Makefile

```diff
@@ -1,3 +1,19 @@
+build_conda_package = \
```

Comment:
Is it possible to create 2 separate make targets (conda-noarch and conda-arch) instead? And maybe a 3rd one for moving stuff around (the last section of the function)